distribution/src/config/opensearch.yml

```diff
@@ -87,6 +87,13 @@ ${path.logs}
 #
 #action.destructive_requires_name: true
 #
+# ---------------------------------- Remote Store -----------------------------------
+# Controls whether cluster imposes index creation only with remote store enabled
+# cluster.remote_store.enabled.force: true
+#
+# Repository to use while enforcing remote store for an index
+# cluster.remote_store.repository.default: my-repo-1
```

Comment:
Don't we need similar settings for translog as well?

Reply:
i was planning to use the same repo for both segment and translog. but we can add a new setting to provide more flexibility.